clustergroup/templates/imperative/job.yaml

```diff
@@ -1,5 +1,5 @@
-{{/* Only define this if there are any imperativejobs defined */}}
-{{- if gt (len $.Values.clusterGroup.imperative.jobs) 0 -}}
+{{/* Define this if needed (jobs defined */}}
```

Comment:
Only if you happen to do another pass for other reasons, it might be nice to align this comment to the one in [clustergroup/templates/imperative/namespace.yaml](https://github.com/hybrid-cloud-patterns/common/pull/130/commits/3ab539360ab91cfd40a09f80cff282e7d6749de2#diff-c10a3be8e6a5d667e38cc2ad48e0bcb27bc5d8f84a887d24be6228e7fc981d0a)